includes/definitions/arista_eos.yaml

```diff
@@ -16,8 +16,10 @@ poller_modules:
     applications: 0
     services: 0
     ntp: 0
+    hardware_utilization: 0
```

Comment:
This is the wrong module, same for discovery below.

Reply:
You've called the module `router-utilization` :)

Reply:
Done, thanks!